src/miniflask/miniflask.py

```diff
@@ -1352,44 +1356,8 @@ def optional(self, variable_type):
         """  # noqa: W291
         return optional_default(variable_type)
 
-    # like with relative imports
-    def like(self, varname, alt, scope="."):
-        r"""
-        Define state dependencies.
-
-        # Note {.alert}
-        Like variables are parsed *after* the CLI-arguments have been parsed:
-        - In case any CLI-argument changes a dependency the like-Dependency should change accordingly.
-        - If any CLI-argument changes the variable defined as a dependency, the dependency is canceled.
-
-        Args:
-        - `varname`: The variable identifier to use for the dependency.
-        - `alt`: The default value if no variable found.
-        - `scope`: The variable scope to search for the variable.
-
-        Examples:
-
-        **Global variables with Fallback**
-        ```python
-        mf.register_defaults({
-            "myvar": mf.like("globalvar", alt=42)
-        })
-        ```
-
-        **Local dependencies**
-        ```python
-        mf.register_defaults({
-            "othervar": mf.like(".var"),
-            "likeparent": mf.like("..parentvar")
-        })
-        ```
-        """  # noqa: W291
-        scope_name = scope
-        if scope is not None:
-            scope, _ = self._get_relative_module_id(scope)
-        return like(varname, alt, scope=scope, scope_name=scope_name)
-
     def as_is_callable(self, variable):
+        # TODO
```

Comment:
Is the TODO part of this this work?

Reply:
Thanks, eagle eye, this is indeed an error.